src/components/Hosts/UpdateStatusModal.tsx

```diff
@@ -84,64 +77,40 @@ export const UpdateStatusModal: React.VFC<Props> = ({
   };
 
   return (
-    <Modal
+    <ConfirmationModal
       data-cy={dataCy}
-      visible={visible}
+      open={visible}
       onCancel={onClickCancel}
+      onConfirm={onClickUpdate}
       title="Update Host Status"
-      footer={[
-        <Button
-          key="modal-cancel-button"
-          data-cy="modal-cancel-button"
-          onClick={onClickCancel}
-        >
-          Cancel
-        </Button>,
-        <LoadingButton
-          key="modal-update-button"
-          data-cy="modal-update-button"
-          variant="primary"
-          disabled={!status}
-          loading={loadingUpdateHostStatus}
-          onClick={onClickUpdate}
-        >
-          Update
-        </LoadingButton>,
-      ]}
+      buttonText="Update"
+      submitDisabled={!status || loadingUpdateHostStatus}
     >
-      <Body weight="medium">Host Status</Body>
-      <StyledSelect
-        aria-labelledby="host-status-select"
+      <Select
+        label="Host Status"
         data-cy="host-status-select"
         value={status}
-        onChange={(s) => setHostStatus(s as UpdateHostStatus)}
+        onChange={(s) => {
+          setHostStatus(s as UpdateHostStatus);
+        }}
       >
         {hostStatuses.map(({ title, value, key }) => (
           <Option key={key} value={value} data-cy={`${value}-option`}>
             {title}
           </Option>
         ))}
-      </StyledSelect>
-      <StyledTextArea
+      </Select>
+      <TextArea
```

Comment:
nit. I think it needs a little more vertical space in between the select and text area

<img width="625" alt="Screen Shot 2022-12-12 at 12 27 44 PM" src="https://user-images.githubusercontent.com/47064971/207112831-9fe4ef94-8c92-451c-828f-e14f0605a908.png">